modules/swagger-codegen/src/main/resources/dart/pubspec.mustache

```diff
@@ -4,7 +4,7 @@ description: {{pubDescription}}
 dependencies:
   http: '>=0.11.1 <0.12.0'
   dartson: "^0.2.4"
-  intl: "^0.12.4+2"
+  intl: any
```

Comment:
@ircecho if the users want to use an older version of the lib, they can always use the customized template via the `-t` option.